pkg/cmd/cmd_import.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 The KCL Authors. All rights reserved.
+
+package cmd
+
+import (
+	"github.com/urfave/cli/v2"
+	"kcl-lang.io/kcl-go/pkg/tools/gen"
+	"kcl-lang.io/kpm/pkg/client"
+	"kcl-lang.io/kpm/pkg/reporter"
+	"os"
+)
+
+// NewImportCmd new a Command for `kpm import`.
+func NewImportCmd(_ *client.KpmClient) *cli.Command {
```

Comment:
`KpmClient` is useful and is mainly responsible for control all functions provided in the kpm cli.